### DISCUSSION TITLE
Logo contest

### DISCUSSION THREAD

**Discussion by ghost** [Ralim/IronOS#819] [Ideas]
Hey @Ralim,
I hope you will forgive the _obsession_ I have with logos, but I think that every project with personality needs a logo that makes it immediately recognizable and worthy of interest.
In this regard, I was thinking of a contest of variable duration (two-three months) open to anyone who feels willing enough to create a graphic for the repo.
Up for grabs a set of Ralim signature tips.
If you like the idea, we could deepen the design, conversely if the idea should seem useless or even stupid, feel free to express your disappointment too. 😄 

Sincerely,
Federico

**Comment by Ralim** [comment]
Heh, I'm not worried at all.

I would love a good logo for the project :)

Tough not sure what "Ralim signature tips" would be dobbing me in for here 😂 .

Having some form of a logo would be quite sweet tbh, especially since the project has a proper name too.

**Reply by ghost** [reply]
I was thinking of a set of tips, either engraved or with an autograph in a possible case, to give a peculiar [sense of uniqueness](http://www.fendercustomshop.com/series/artist/).
In this regard, donation funds could be diverted to finance the purchase of the tips and shipping costs, also emphasizing the possibility of making a donation (up to 10-20 $: higher donations, albeit in good faith, could _distort_ the sense of the initiative).
Alternatively, a special fund could be set up to start the initiative.
Clearly the iron model in use by the participant should be requested in the participation form.

Thoughts? :)
